src/components/text-input/TextInput.scss

```diff
@@ -1,55 +1,105 @@
 @import '../../styles/variables';
+@import '../../styles/inputs';
 
-/**************************************
- * Text Input
- **************************************/
+.bdl-TextInput-input {
```

Comment:
Are we using `.bdl-` as the new standard prefix for common styles? Are we using BEM going forward, as well, with `TextInput` being BookCase?

Reply:
This example is following SUIT css style. Still TBD if we will use SUIT vs BEM.